<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkinDialog</class>
 <widget class="QDialog" name="SkinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Upload A New Skin</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="layoutForm">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPath">
       <property name="text">
        <string>Skin file</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="layoutPath">
       <item>
        <widget class="QLineEdit" name="pathEdit">
         <property name="placeholderText">
          <string>Path to the skin file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonBrowse">
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkBoxSlim">
       <property name="text">
        <string>Skin for the &quot;slim&quot; player model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vspc">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutDown">
     <item>
      <spacer name="hspc">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUpload">
       <property name="text">
        <string>Upload</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBrowse</tabstop>
  <tabstop>checkBoxSlim</tabstop>
  <tabstop>buttonUpload</tabstop>
  <tabstop>pathEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
